src/hooks/useRankingData.js

```diff
@@ -113,14 +118,50 @@ const useRankingData = () => {
       state: { email, name, photo },
     });
 
+  const onChangeSelectedRanking = newSelectedRanking => {
+    if (newSelectedRanking !== selectedRanking) {
+      setSelectedRanking(newSelectedRanking);
+      var newRankingData = [...rankingData];
```

Comment:
Why `var`?

Reply:
because Im assigning it later

![image](https://user-images.githubusercontent.com/8755889/209966019-4e31bdc7-3bda-4d1f-9a52-b7b206c6ffd3.png)

Reply:
because I want to see the replay and check if it was offside or not

Reply:
😂 😂 👏